.npmignore

```diff
@@ -1,6 +1,3 @@
 .tscache
 node_modules
-dist
```

Comment:
You're right I should've left this line so TS output doesn't get published to npm. The next two lines can still be removed though and maybe add `dist.babel/test`. I didn't change `.gitignore` though.